ansible_collections/arista/avd/molecule/eos_cli_config_gen_v4.0/inventory/host_vars/sflow.yml

```diff
@@ -1,25 +1,25 @@
 ### Sflow ###
 sflow:
   vrfs:
-    MGMT:
+    - name: MGMT
       destinations:
-        10.6.75.62:
+        - sflow_destination_ip: 10.6.75.62
```

Comment:
this can be IP, ipv6 or FQDN, so I think we should just say `destination`

Reply:
Updated!